app/controllers/feature_flags_controller.rb

```diff
@@ -0,0 +1,34 @@
+class FeatureFlagsController < ApplicationController
```

Comment:
This was used to allow people to turn the GAI feature flag on/off during testing. Making a note to remove this before we merge in.